docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -45,16 +45,17 @@ export default async function auth(req, res) {
   const response = await authorize({
     room,
     secret,
-    userId: "123", // Optional
+// Optional, corresponds to the UserMeta.id type defined in liveblocks.config.ts
```

Comment:
```suggestion
// Optional, corresponds to the UserMeta["id"] type defined in liveblocks.config.ts
```

Do you think we should change all these to bracket notation? Dot-notation isn't supported in TypeScript, and we're talking about TypeScript types here